SPECS/mariner-rpm-macros/mariner-rpm-macros.spec

```
@@ -79,7 +87,9 @@ install -p -m 755 -t %{buildroot}%{rcdir} gen-ld-script.sh
 install -p -m 755 -t %{buildroot}%{rcdir} generate-package-note.py
 install -p -m 755 -t %{buildroot}%{rcdir} verify-package-notes.sh
 
-sed -e 's|@DIST@|%{dist}|g' %{SOURCE28} > macros.dist
+major_version=`echo %{mariner_release_version} | cut -d'.' -f1`
+sed -e 's|@DIST@|%{dist}|g' %{SOURCE28} | sed -e "s|@MAJORVER@|$major_version|g" > macros.dist
```

Just an idea to keep things shorter, not an actual issue:
```suggestion
sed -e 's|@DIST@|%{dist}|g' -e "s|@MAJORVER@|$major_version|g" %{SOURCE28} > macros.dist
```